docs/features/scales.md

```diff
@@ -949,6 +949,7 @@ Plot automatically generates [axis](../marks/axis.md) and optionally [grid](../m
 * **label** - a string to label the axis
 * **labelAnchor** - the label anchor: *top*, *right*, *bottom*, *left*, or *center*
 * **labelOffset** - the label position offset (in pixels; default depends on margins and orientation)
+* **labelArrow** - the label arrow: *left*, *right*, *top* or *bottom*; *none*, or null, or false to suppress; true to indicate the scale’s ascending order; defaults to true for quantitative and temporal scales, false for ordinal scales
```

Comment:
It’s up, right, down, left instead of top, right, bottom, left (because it’s a direction rather than a side).

Reply:
Also, this is missing *auto*, which is the default value.